methods/dst.py

```diff
@@ -41,15 +44,15 @@
 
 
 class DST(DetectionMethod):
-    def run(self, url, screenshot_url, uuid, pagetitle, image64 = "") -> ResultType:
+    async def run(self, url, screenshot_url, uuid, pagetitle, image64 = "") -> ResultType:
```

Comment:
If you want to make the method of the subclass async you should also make that of the base class async, since it changes the signature